python/batch-predictor/docker/process_conda_env.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This script is designed to manipulate a Conda environment file to add Merlin dependencies.
+#
+# Usage:
+# ./process_conda_env.sh CONDA_ENV_PATH MERLIN_DEP MERLIN_DEP_CONSTRAINT
+#
+# Input:
+# CONDA_ENV_PATH: Path to the Conda environment YAML file.
+# MERLIN_DEP: The dependency to be added or removed.
```

Comment:
at first i thought even if user specific it will be removed/added, but i tested it all fine 👍 

![image](https://github.com/caraml-dev/merlin/assets/30390872/89a5b432-197f-40c4-858f-abd23d41bc76)